scripts/runfabtests.sh

```diff
@@ -187,21 +182,23 @@ function cs_test {
 	fi
 }
 
-function main {
-	local -r tests=$(echo $1 | sed 's/all/unit,simple,quick,standard/' | tr ',' ' ')
+function main {	
+	if [[ $1 == "quick" ]]; then
+		local -r tests=$(echo $1 | sed 's/quick/unit,simple,quick/' | tr ',' ' ')
```

Comment:
We should rename the quick test set to differentiate it from the quick meta-set. Maybe short?

Right now there is no way to run the just the quick_tests.